pkgs/servers/monitoring/lcdproc/default.nix

```
@@ -39,11 +59,11 @@ stdenv.mkDerivation rec {
       --replace server/drivers/ $out/lib/lcdproc/
   '';
 
-  meta = with lib; {
+  meta = with stdenv.lib; {
```

That won't work anymore.

```suggestion
  meta = with lib; {
```